rlclientlib/azure_factories.cc

```diff
@@ -31,16 +33,34 @@ namespace reinforcement_learning {
   }
 
   int restapi_data_transport_create(m::i_data_transport** retval, const u::configuration& config, i_trace* trace_logger, api_status* status) {
-    const auto uri = config.get(name::MODEL_BLOB_URI, nullptr);
-    if (uri == nullptr) {
+    const auto model_uri = config.get(name::MODEL_BLOB_URI, nullptr);
+    if (model_uri == nullptr) {
       RETURN_ERROR(trace_logger, status, http_uri_not_provided);
     }
     i_http_client* client;
-    RETURN_IF_FAIL(create_http_client(uri, config, &client, status));
+    RETURN_IF_FAIL(create_http_client(model_uri, config, &client, status));
     *retval = new m::restapi_data_transport(client, trace_logger);
     return error_code::success;
   }
 
+  int restapi_endpoint_data_transport_create(m::i_data_transport** retval, const u::configuration& config, i_trace* trace_logger, api_status* status) {
```

Comment:
This function naming doesn't really make sense. `restapi_data_transport_create` and `restapi_endpoint_data_transport_create` are both restapi endpoints. What you're trying to distinguish is `restapi` vs `authenticated restapi`.

Reply:
renamed function name